crates/db-address-space/README.md

```diff
@@ -0,0 +1,41 @@
+# db-address-space
+
+## Design
+
+db-address-space crate is designed as the address space of guest memory. We define several data structures in this crate to describe logical information about address space of guest memory for VMM to use.
```

Comment:
The `db-address-space` crate is an address space manager for virtual machines, which manages memory and MMIO resources resident in the guest physical address space.